contracts/upgradeable_contracts/components/common/TokensRelayer.sol

```diff
@@ -31,8 +31,6 @@ abstract contract TokensRelayer is BasicAMBMediator, ReentrancyGuard {
                 assembly {
                     receiver := calldataload(120)
                 }
-                require(receiver != address(0));
-                require(receiver != mediatorContractOnOtherSide());
```

Comment:
Why did you remove this?

Reply:
I have found a duplicated check in the `bridgeSpecificActionsOnTokenTransfer()` in both HomeOmnibridge and ForeignOmnibridge contracts.

Reply:
ok. thanks